devops/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+﻿vagrant plugin install sahara
```

Comment:
I foresee a package called vagrant-sahara that this would take a dependency on.

Reply:
@mkevenaar Said he would create this package.